sdk/src/tx/reportTransactionError.ts

```diff
@@ -0,0 +1,96 @@
+import { Connection, SendTransactionError, VersionedTransactionResponse } from "@solana/web3.js";
+
+const getTransactionResult = async (txSig: string, connection: Connection): Promise<VersionedTransactionResponse> => {
+    return await connection.getTransaction(txSig, {
+        maxSupportedTransactionVersion: 0,
```

Comment:
might want to specified 'confirmed' commitment here, im not sure what the default is